tools/topology/topology2/cavs/platform/intel/hdmi-generic.conf

```diff
@@ -0,0 +1,193 @@
+Object.Dai {
+	HDA.4 {
+		name iDisp1
+		id $HDMI1_ID
+		default_hw_conf_id 1
+		Object.Base.hw_config.HDA4 {}
+		direction playback
+	}
+	HDA.5 {
+		name iDisp2
+		id $HDMI2_ID
+		default_hw_conf_id 2
+		Object.Base.hw_config.HDA5 {}
+		direction playback
+	}
+	HDA.6 {
+		name iDisp3
+		id $HDMI3_ID
+		default_hw_conf_id 3
+		Object.Base.hw_config.HDA6 {}
+		direction playback
+	}
+	HDA.7 {
+		name iDisp4
+		id $HDMI4_ID
+		default_hw_conf_id 3
+		Object.Base.hw_config.HDA6 {}
+		direction playback
+	}
+}
+Object.Pipeline {
+	passthrough-playback.20 {
+		Object.Widget.pipeline.1 {
+			stream_name $HDMI1_PIPELINE_STREAM_NAME
+		}
+		Object.Widget.copier.1 {
+			stream_name $HDMI1_PCM_CAPS
+		}
+
+		format s16le
```

Comment:
?? why s16le for HDMI?

Reply:
> ?? why s16le for HDMI?

It was copied from the existing https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/cavs/cavs-passthrough-hdmi.conf

Reply:
It makes no sense, we never had any restriction to s16 for topology1... I also don't really get what 'passthrough' means for HDMI, because we still have copiers that are capable of adjusting formats, no?